internal/to0/to0.go

```diff
@@ -24,32 +24,32 @@ type to0Client interface {
 // Allow test-time injection of dependencies.
 var (
 	newTO0Client = func(vouchers fdo.OwnerVoucherPersistentState, keys fdo.OwnerKeyPersistentState) to0Client {
-		return &fdo.TO0Client{Vouchers: vouchers, OwnerKeys: keys}
+		return &fdo.TO0Client{Vouchers: vouchers, OwnerKeys: keys, TTL: 300}
```

Comment:
This is hardcoded in owner.go as well - probably should be passed from owner.go via RegisterRvBlob since the retry loop also uses it. Would be easy to surface as a config item at some point if asked.